xclim/indicators/land/_snow.py

```diff
@@ -13,35 +12,22 @@
 ]
 
 
-class SnowDepth(Daily):
-    cfcheck = staticmethod(generate_cfcheck("snd"))
+# We need to declare the base class here so the `land` module is detected automatically.
```

Comment:
Why is this necessary?

Reply:
To remove the need of passing `module='land'` and `realm='land'`, the Indicator's `__new__` checks where the class was declared (with the default `module`).  Because we don't know where instances are created, only where classes are (explicitly) defined.